### PR TITLE
1.3-beta6

### PR DESCRIPTION
**Pull request by slaxor505**
Please merge.

### CODE REVIEW

File: custom_components/openplantbook/uploader.py [hunk 1]
```diff
@@ -33,6 +33,87 @@
...
+        if unit_of_measurement == "°F":
+            supported_state = round((supported_state - 32) * 5 / 9)
+            _LOGGER.debug(
+                "Temperature converted from %s °F to %s °C"
+                % (state.state, supported_state)
+            )
+            unit_of_measurement = "°C"
+
+        # Convert Kelvin to Celsius
+        elif unit_of_measurement == "K":
```
Comment (Olen) [inline custom_components/openplantbook/uploader.py:83]: Should this be "K" or °K"?

Comment (slaxor505) [reply custom_components/openplantbook/uploader.py:83]: Thanks for reviewing the code, mate. It is just "K" as per:

https://github.com/home-assistant/core/blob/be5577c2f9486167696aa7920cdb9e342cda2bf7/homeassistant/const.py
`
# Temperature units
class UnitOfTemperature(StrEnum):
    """Temperature units."""

    CELSIUS = "°C"
    FAHRENHEIT = "°F"
    KELVIN = "K"
`

**Comment by slaxor505** [comment]
I have just refactored this to HASS constants to be perfect :)

**Comment by slaxor505** [comment]
Just hold on merging it. It seems that there is an issue.

**Comment by slaxor505** [comment]
I fixed the issue within  json_timeseries library. Please merge.

**Comment by slaxor505** [comment]
Hey @Olen . Just wanted to follow-up on this PR. Can you please merge as soon as possible? Thank you.

**Pull request merged**